src/stories/containers/TransparencyReport/components/ExpenseReport/ExpenseReport.tsx

```diff
@@ -29,16 +32,11 @@ interface ExpenseReportProps {
   budgetStatements?: BudgetStatementDto[];
   code: string;
   longCode: string;
-  expenseCategories: ExpenseCategory[];
 }
 
-const ExpenseReport: React.FC<ExpenseReportProps> = ({
-  currentMonth,
-  budgetStatements,
-  code,
-  longCode,
-  expenseCategories,
-}) => {
+const ExpenseReport: React.FC<ExpenseReportProps> = ({ currentMonth, budgetStatements, code, longCode }) => {
+  const { checkOut, handleChangeItemAccordion, handleCheckedExpandedAll, headCountCategories, noHeadCountCategories } =
+    useCategoriesContextModal();
```

Comment:
Note that once you move the invocation of this hook to inside the modal, this is no needed at all here

Reply:
done